docs/reference-guide/backup.md

```diff
@@ -131,6 +131,27 @@ To restore from back-up:
     docker run -p 7687:7687 -p 7444:7444 -v ~/snapshots:/var/lib/memgraph/snapshots memgraph/memgraph
     ```
 
+2. Copy the backed-up snapshot file into the `snapshots` directory after creating the container and start the database. So the commands should look like this: 
+
+    ```
+    docker create -p 7687:7687 -p 7444:7444 -v `snapshots`:/var/lib/memgraph/snapshots --name memgraphDB memgraph/memgraph
+    tar -cf - sample_snapshot_file | docker cp -a  - memgraphDB:/var/lib/memgraph/snapshots
+    ```
+    Here the `sample_snapshot_file` the snapshot file you want to use to restore the data. Due to the nature of the docker ownership of the files, you need to use `tar` to copy the file as STDIN into the non-running container. This way you can give ownership of the file to the `memgraph` user inside the container.
```

Comment:
```suggestion
    docker create -p 7687:7687 -p 7444:7444 -v `snapshots`:/var/lib/memgraph/snapshots --name memgraphDB memgraph/memgraph
    tar -cf - sample_snapshot_file | docker cp -a - memgraphDB:/var/lib/memgraph/snapshots
    ```
    The `sample_snapshot_file` is the snapshot file you want to use to restore the data. Due to the nature of Docker file ownership, you need to use `tar` to copy the file as STDIN into the non-running container. It will allow you to change the ownership of the file to the `memgraph` user inside the container.
```